packages/fx-core/src/component/generator/copilotExtension/generator.ts

```diff
@@ -52,12 +50,12 @@ const enum telemetryProperties {
 export class CopilotExtensionGenerator extends DefaultTemplateGenerator {
   componentName = "copilot-extension-from-scratch-generator";
   public activate(context: Context, inputs: Inputs): boolean {
-    return (
-      (inputs[QuestionNames.Capabilities] === CapabilityOptions.declarativeAgent().id &&
-        inputs[QuestionNames.ApiPluginType] !== ApiPluginStartOptions.apiSpec().id) ||
-      (inputs[QuestionNames.Capabilities] === CapabilityOptions.apiPlugin().id &&
-        inputs[QuestionNames.ApiPluginType] === ApiPluginStartOptions.newApi().id)
-    );
+    return [
+      TemplateNames.ApiPluginFromScratch,
+      TemplateNames.ApiPluginFromScratchBearer,
```

Comment:
The ApiPluginFromScratchBearer template is currently actually ApiPluginFromScratchAPIKey. We can add a TODO to update the name in the future.

cc: @huimiu